### ISSUE TITLE
Failed to open registry repository at \\?\C:\Users\<redacted>\.cargo\registry\index\github.com-1ecc6299db9ec823

### ISSUE THREAD

**Issue by CBenoit** [nabijaczleweli/cargo-update#203]
Hello!

When I try to run `cargo install-update --all` (or `--list`)  on Windows (11), I get the following error:

```
Failed to open registry repository at \\?\C:\Users\<redacted>\.cargo\registry\index\github.com-1ecc6299db9ec823.
```

**Comment by nabijaczleweli** [comment]
uuuuh. mistakes of the youth to not include the error; can you try the current `master` branch (at least 8d12e222b1a8d99a20450e399f056cc10e97d475)?

**Comment by CBenoit** [comment]
Haha, I understand! Sure thing! Here is the new error I get:

> Failed to open registry repository at \\?\C:\Users\<redacted>\.cargo\registry\index\github.com-1ecc6299db9ec823: failed to parse config file: invalid configuration key (in C:/Users/<redacted>/.gitconfig:103); class=Config (7).

Luckily, I already know this error 😄 

This is due to this config block from my `.gitconfig` file:

> [delta "zebra-dark"]
    map-styles = \
       bold purple => syntax "#380233", \
       bold blue => syntax "#1b0738", \
       bold cyan => syntax "#00302b", \
       bold yellow => syntax "#222f14"

I got it with other Rust-written CLI tools in the past. This is due to `libgit2` dependency being outdated. Indeed, I see `cargo-update` is depending on the crate `git2:0.12` but the latest is `git2:0.15`. I can send a PR for that =)

**Comment by nabijaczleweli** [comment]
Released in [`v11.0.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v11.0.0)
